fortanix-vme/aws-nitro-enclaves/nitro-attestation-verify/Cargo.toml

```diff
@@ -10,7 +10,7 @@ chrono = "0.4"
 serde_cbor = "0.11"
 # Required until PR36 is accepted
```

Comment:
These two comments can be cleaned up now since we are not using the branch here anymore.

Reply:
I missed that. Good catch! This patch for v0.5 isn't even needed.